_codux/ui-kit/components/components.board.tsx

```diff
@@ -2,69 +2,96 @@ import { createBoard, Variant } from '@wixc3/react-board';
-                        ribbon="NEW"
-                        price="$6"
-                        discountedPrice="$5.5"
-                    />
-                </Variant>
-            </div>
+            <Kit category="Core Components" title="Components & Elements">
+                <Kit.Section title="Input">
+                    <Kit.Item>
+                        <Variant name="Number Input">
+                            <QuantityInput value={6} onChange={() => {}} />
+                        </Variant>
+                        <Kit.Description>Number Input</Kit.Description>
+                    </Kit.Item>
+                </Kit.Section>
+
+                <Kit.Section title="Accordion">
+                    <Kit.Item>
+                        <Variant name="Accordion">
+                            <Accordion
+                                items={[
+                                    {
+                                        title: 'Product Info',
+                                        content: 'Content',
+                                    },
+                                    {
+                                        title: 'Return & Refund Policy',
+                                        content: 'Content',
+                                    },
+                                    {
+                                        title: 'Shipping Info ',
+                                        content: 'Content',
+                                    },
+                                ]}
+                                className={styles.demoWidth}
+                            />
+                        </Variant>
+                        <Kit.Description>Accordion</Kit.Description>
+                    </Kit.Item>
+                </Kit.Section>
+
+                <Kit.Section title="Labels">
+                    <Kit.Item>
+                        <Variant name="Ribbon">
+                            <span className="ribbon">Sale</span>
+                        </Variant>
+                        <Kit.Description>Ribbon</Kit.Description>
+                    </Kit.Item>
+                </Kit.Section>
+
+                <Kit.Section title="Cards">
+                    <Kit.Item className={styles.demoWidth}>
+                        <Variant name="Product Card">
+                            <ProductCard
+                                name="Bamboo Toothbrush"
+                                imageUrl="https://static.wixstatic.com/media/c837a6_18152edaef9940ca88f446ae94b48a47~mv2.jpg/v1/fill/w_824,h_1098,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/c837a6_18152edaef9940ca88f446ae94b48a47~mv2.jpg"
+                                ribbon="NEW"
+                                price="$6"
+                                discountedPrice="$5.5"
+                            />
+                        </Variant>
+                        <Kit.Description>Product Card</Kit.Description>
```

Comment:
It looks as if "Product Card" is part of the card. Can we put the kit description on top and with a distinctive styling?

<img width="408" alt="Screenshot 2024-09-30 at 14 28 04" src="https://github.com/user-attachments/assets/79e47155-0eb9-4dab-981b-008372f69ff1">

IMO it's doesn't look good in sections too.

<img width="1077" alt="Screenshot 2024-09-30 at 14 30 42" src="https://github.com/user-attachments/assets/984d6969-c47c-40a0-968d-fd8be77578f3">

Reply:
If there's only one component under the section, we can remove the description,
if there's more than one, we keep it.  

Note: This is a temporary solution - @RanBerkovich1  has a task that gives each variant a name automatically